### PR TITLE
Update 06_Mobile_access.md

### LINKED ISSUE #4314: docs refer to numbered steps but steps aren't numbered.
On this page https://github.com/FreshRSS/FreshRSS/blob/master/docs/en/users/06_Mobile_access.md (accessed via https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html), the text refer to numbered steps, e.g. "Proceed to step 5."

However there are only 2 numbered items in the page. Maybe the list formatting got changed at some point?

Might make sense to just used named subheadings to refer to different sections because that way a reader can figure out what they are looking for, but I don't know your workflow.

### PR DESCRIPTION
**Pull request by math-GH**
Closes #4314

Old steps see:
https://github.com/FreshRSS/FreshRSS/blob/4fa6e273979e13361a769d1b1a9ecfc4948f5570/docs/en/users/06_Mobile_access.md

### CODE REVIEW

File: docs/en/users/06_Mobile_access.md [hunk 1]
```diff
@@ -32,18 +32,18 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
...
 
 ### Fix server configuration
 
 * Click on the second link “Check partial server configuration (without `%2F` support)”:
 	* If you get `PASS`, then the problem is indeed that your server does not accept slashes `/` that are escaped `%2F`.
 		* With Apache, remember the directive [`AllowEncodedSlashes On`](http://httpd.apache.org/docs/trunk/mod/core.html#allowencodedslashes)
-		* Or use a client that does not escape slashes (such as EasyRSS), in which case proceed to step 6.
-	* If you get *Service Unavailable!*, then check from step 1 again.
+		* Or use a client that does not escape slashes (such as EasyRSS) ([`check client list`](https://github.com/FreshRSS/FreshRSS#apis--native-apps)).
+	* If you get *Service Unavailable!*, then check from above chapter "Enable the API in FreshRSS" again.
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* If you get *Service Unavailable!*, then check the preceding section "Enable the API in FreshRSS" again.
```

File: docs/en/users/06_Mobile_access.md [hunk 2]
```diff
@@ -55,4 +55,4 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
 		* If you get *File not found!*, check your server `fastcgi_split_path_info`.
 	* If you get *FAIL 64-bit or GMP extension!*, then your PHP version does not pass the requirement of being 64-bit and/or have PHP [GMP](http://php.net/gmp) extension.
 		* The easiest is to add the GMP extension. On Debian / Ubuntu: `sudo apt install php-gmp`
-	* Update and try again from step 3.
+	* Update and try again from above chapter "Testing".
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* Update and try again from the preceding section "Testing".
```

File: docs/en/users/06_Mobile_access.md [hunk 3]
```diff
@@ -32,18 +32,18 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
 
 1. Under the section “Profile”, click on the link like `https://rss.example.net/api/` next to the field “API password”.
 2. Click on first link “Check full server configuration”:
-	* If you get *PASS* then you are done, all is good: you may proceed to step 6.
-	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`. Proceed to step 5.
-	* If you get any other error message, proceed to step 5.
+	* If you get *PASS* then you are done, all is good.
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* If you get *PASS* then you are done; all is well.
```

File: docs/en/users/06_Mobile_access.md [hunk 4]
```diff
@@ -32,18 +32,18 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
...
+	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see next chapter "Fix server configuration".
+	* If you get any other error message, see next chapter "Fix server configuration".
```
Comment (Frenzie) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see the next section "Fix server configuration".
	* If you receive any other error message, see the next section "Fix server configuration".
```

**Review by Frenzie — COMMENTED** [review]
Looks good!

File: docs/en/users/06_Mobile_access.md [hunk 5]
```diff
@@ -55,4 +54,4 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
 		* If you get *File not found!*, check your server `fastcgi_split_path_info`.
 	* If you get *FAIL 64-bit or GMP extension!*, then your PHP version does not pass the requirement of being 64-bit and/or have PHP [GMP](http://php.net/gmp) extension.
 		* The easiest is to add the GMP extension. On Debian / Ubuntu: `sudo apt install php-gmp`
-	* Update and try again from step 3.
+	* Update and try again from the preceding section "Testing".
```
Comment (Alkarex) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* Update and try again from the preceding section “Testing”.
```

Minor quotes

File: docs/en/users/06_Mobile_access.md [hunk 6]
```diff
@@ -32,18 +32,17 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
...
 
 ### Fix server configuration
 
 * Click on the second link “Check partial server configuration (without `%2F` support)”:
 	* If you get `PASS`, then the problem is indeed that your server does not accept slashes `/` that are escaped `%2F`.
 		* With Apache, remember the directive [`AllowEncodedSlashes On`](http://httpd.apache.org/docs/trunk/mod/core.html#allowencodedslashes)
-		* Or use a client that does not escape slashes (such as EasyRSS), in which case proceed to step 6.
-	* If you get *Service Unavailable!*, then check from step 1 again.
+		* Or use a client that does not escape slashes (such as EasyRSS) ([`check client list`](https://github.com/FreshRSS/FreshRSS#apis--native-apps)).
+	* If you get *Service Unavailable!*, then check the preceding section "Enable the API in FreshRSS" again.
```
Comment (Alkarex) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* If you get *Service Unavailable!*, then check the preceding section “Enable the API in FreshRSS” again.
```

File: docs/en/users/06_Mobile_access.md [hunk 7]
```diff
@@ -32,18 +32,17 @@ See the [page about the Fever compatible API](../developers/06_Fever_API.md) for
 
 1. Under the section “Profile”, click on the link like `https://rss.example.net/api/` next to the field “API password”.
 2. Click on first link “Check full server configuration”:
-	* If you get *PASS* then you are done, all is good: you may proceed to step 6.
-	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`. Proceed to step 5.
-	* If you get any other error message, proceed to step 5.
-
+	* If you get *PASS* then you are done; all is well.
+	* If you get *Bad Request!* or *Not Found*, then your server probably does not accept slashes `/` that are escaped `%2F`, see the next section "Fix server configuration".
+	* If you receive any other error message, see the next section "Fix server configuration".
```
Comment (Alkarex) [inline docs/en/users/06_Mobile_access.md]:
```suggestion
	* If you receive any other error message, see the next section “Fix server configuration”.
```

Either typographic quotes or *`*`emphasis`*`*

**Review by Alkarex — APPROVED** [review]

**Comment by Frenzie** [comment]
I'd go with emphasis. :)

**Pull request merged**